src/connector/src/sink/encoder/mod.rs

```diff
@@ -131,3 +131,8 @@ impl From<FieldEncodeError> for super::SinkError {
         Self::Encode(value.to_string())
     }
 }
+
+#[derive(Clone)]
+pub struct KafkaConnectParams {
+    pub schema_name: String,
+}
```

Comment:
if you do not want to edit the member, you may use `KafkaConectParamsRef = Arc<KafkaConnectParams>`